pkg/guacrest/server/retrieveDependencies_test.go

```diff
@@ -583,50 +469,45 @@ func Test_ClientErrors(t *testing.T) {
 			Ingest(ctx, t, gqlClient, tt.data)
 			restApi := server.NewDefaultServer(gqlClient)
 
-			res, err := restApi.RetrieveDependencies(ctx, api.RetrieveDependenciesRequestObject{Params: tt.input})
+			res, err := restApi.GetPackageDeps(ctx, gen.GetPackageDepsRequestObject{Purl: tt.purl})
 			if err != nil {
 				t.Fatalf("RetrieveDependencies returned unexpected error: %v", err)
 			}
-			if _, ok := res.(api.RetrieveDependencies400JSONResponse); !ok {
+			if _, ok := res.(gen.GetPackageDeps400JSONResponse); !ok {
 				t.Fatalf("Did not receive a 400 Response: recieved %v of type %T", res, res)
 			}
 
 		})
 	}
 }
 
-func Test_DefaultLinkCondition(t *testing.T) {
-	/******** set up the test ********/
+func Test_ClientErrorsForArtifact(t *testing.T) {
```

Comment:
There should be a test for a badly formatted input, such as missing the hash algorithm - which I don't think is implemented.

Reply:
Yeah, I have now added a missing algorithm prefix test case.